ts/test/helpers/protobuf-validation.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Generic test helpers for protobuf deserialization validation
+ */
+import { expect } from "@jest/globals";
+import Long from "long";
+
+/**
+ * Generic function to validate protobuf deserialization for ANY type
+ * 
+ * Usage examples:
+ *   validateProtobufDeserialization<QueryCertificatesResponse>(certificateResponse);
+ *   validateProtobufDeserialization<QueryLeaseResponse>(marketResponse);
+ *   validateProtobufDeserialization<YourProtobufType>(anyProtobufObject);
+ * 
+ * Benefits:
+ *   - Full TypeScript type safety and IntelliSense after validation
+ *   - Works with any protobuf-generated type
+ *   - No need to specify field names or structures manually
+ *   - Uses TypeScript assertion to provide type safety after validation
+ */
+export function validateProtobufDeserialization<T>(obj: any): asserts obj is T {
```

Comment:
I don't think it makes sense to have something like this because the main stuff which I wanted to check in serialization is handling of custom type. Especially magic stuff around `LegacyDec` which has different representation on the wire and in runtime. Everything else should be pretty much reliable.

But this function won't find this discrepancy in LegacyDec type handling between GoLang and Nodejs. because both will use string but the number inside may be different. The same applies to other protobuf types which have `(customtype)` attribute